### PR TITLE
Apply light edits to 8.0 changelog

### PR DESCRIPTION
**Pull request by dedemorton**
Saw a couple typos, so I made some light edits.

### CODE REVIEW

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -12,47 +12,46 @@ https://github.com/elastic/beats/compare/v7.17.0...v8.0.0[View commits]
...
 - Improve stats API by adding host metadata. {pull}27963[27963]
 - Libbeat: logp package forces ECS compliant logs. Logs are JSON formatted. Options to enable ECS/JSON have been removed. {issue}15544[15544] {pull}28573[28573]
 - Remove `auto` from the available options of `setup.ilm.enabled` and set the default value to `true`. {pull}28671[28671]
 - add_process_metadata processor: Replace usage of deprecated `process.ppid` field with `process.parent.pid`. {pull}28620[28620]
 - add_docker_metadata processor: Replace usage of deprecated `process.ppid` field with `process.parent.pid`. {pull}28620[28620]
 - Use data streams instead of indices for storing events from Beats. {pull}28450[28450]
 - Remove option `setup.template.type` and always load composable template with data streams. {pull}28450[28450]
-- Remove several ILM options (`rollover_alias` and `pattern`) as data streams does not require index aliases. {pull}28450[28450]
-- Index template's default_fields setting is only populated with ECS fields. {pull}28596[28596] {issue}28215[28215]
-- Remove `auto` from the available options of `setup.ilm.enabled` and set the default value to `true`. {pull}28671[28671]
```
Comment (dedemorton) [inline CHANGELOG.asciidoc:27]: Removed because this is a dupe of l20

File: CHANGELOG.asciidoc [hunk 2]
```diff
@@ -12,47 +12,46 @@ https://github.com/elastic/beats/compare/v7.17.0...v8.0.0[View commits]
...
+- Populate index template's `default_fields` setting with ECS fields only. {pull}28596[28596] {issue}28215[28215]
 - Remove deprecated `--template` and `--ilm-policy` flags. Use `--index-management` instead. {pull}28870[28870]
-- Remove options `logging.files.suffix` and default to datetime endings in log file names. The format of the new name is `{beatname}-{date}(-n)?.ndjson`. Exmaple log files names from oldest to newest: `filebeat-20200101.ndjson`, `filebeat-20200101-1.ndjson`, `filebeat-20200101-2.ndjson`. {pull}28927[28927]
-- Allign kubernetes configuration settings. {pull}29908[29908]
-- The extension of the log files of Beats and Elastic Agent is changed to `.ndjson`. If you are collecting the logs, you must change the path configuration to `/path/to/logs/{beatname}*.ndjson` to avoid any issues. {pull}28927[28927]
+- Remove `logging.files.suffix` option, and default to datetime endings in log file names. The format of the new name is `{beatname}-{date}(-n)?.ndjson`. Example log files names from oldest to newest: `filebeat-20200101.ndjson`, `filebeat-20200101-1.ndjson`, `filebeat-20200101-2.ndjson`. {pull}28927[28927]
```
Comment (dedemorton) [inline CHANGELOG.asciidoc:28]:
```suggestion
- Remove `logging.files.suffix` option, and default to datetime endings in log file names. The format of the new name is `{beatname}-{date}(-n)?.ndjson`. Example log file names from oldest to newest: `filebeat-20200101.ndjson`, `filebeat-20200101-1.ndjson`, `filebeat-20200101-2.ndjson`. {pull}28927[28927]
```

Comment (dedemorton) [inline CHANGELOG.asciidoc]:
```suggestion
- Remove `logging.files.suffix` option, and default to datetime endings in log file names. The format of the new name is `{beatname}-{date}(-n)?.ndjson`. Example log file names from oldest to newest: `filebeat-20200101.ndjson`, `filebeat-20200101-1.ndjson`, `filebeat-20200101-2.ndjson`. {pull}28927[28927]
```

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -84,10 +83,10 @@ https://github.com/elastic/beats/compare/v7.17.0...v8.0.0[View commits]
...
-- Add config option `rotate_on_startup` to file output {issue}19150[19150] {pull}19347[19347]
+- Add config option `rotate_on_startup` to file output. {issue}19150[19150] {pull}19347[19347]
 - Update to ECS 8.0 fields. {pull}28620[28620]
-- Support custom analyzers in fields.yml. {issue}28540[28540] {pull}28926[28926]
-- Support self signed certificates on outputs {pull}29229[29229]
+- Support custom analyzers in `fields.yml`. {issue}28540[28540] {pull}28926[28926]
+- Support self-signed certificates on outputs. {pull}29229[29229]
 - Add FIPS configuration option for all AWS API calls. {pull}[28899]
 - Warn users when connecting to older versions of Elasticsearch instances. {pull}29723[29723]
 - `add_fields` processor is now able to set metadata in events {pull}30092[30092]
```
Comment (bmorelli25) [inline CHANGELOG.asciidoc:92]:
```suggestion
- `add_fields` processor is now able to set metadata in events. {pull}30092[30092]
```

Comment (bmorelli25) [reply CHANGELOG.asciidoc:92]: Found the only one without a period 😄

**Review by bmorelli25 — APPROVED** [review]

**Comment by bmorelli25** [comment]
Should this go to 8.1 too?

**Pull request merged**